js/Domupdates.js

```diff
@@ -0,0 +1,87 @@
+// start with event listner for start button that triggers the start of the game, dommanipulation method for putting names on the screen, generates the puzzle and populates it on the screen. Intialize game function.
+
+const player1 = document.querySelector('.player1-name-input');
```

Comment:
I wonder if some of these global variables should live in index.js. We should ask the instructors to explain the real difference.

Reply:
Sounds like we should merge some of the non-dom manipulation stuff out of this file and back in index.js.